templates/intro-services/README.ipynb

```diff
@@ -0,0 +1,287 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Services\n",
+    "This tutorial shows you how to:\n",
+    "1. Develop a simple Ray Serve application locally.\n",
+    "2. Deploy the application to production as an Anyscale Service.\n",
+    "3. Monitor the production application.\n",
+    "4. Configure service scaling.\n",
+    "\n",
+    "**Note**: This tutorial is run within a Workspace. Please overview the `Introduction to Workspaces` template first before this tutorial."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Develop a Serve app locally\n",
+    "\n",
+    " The fastest way to develop a Ray Serve app is locally within the Workspace. A Serve app running within a workspace behaves identically to a Serve app running as a production service, only it does not have a stable DNS name or fault tolerance.\n",
+    "\n",
+    " To get started, create a file called `my_app.py` and fill it with the following skeleton code:\n",
+    "\n",
+    "```python\n",
+    "import requests\n",
+    "from fastapi import FastAPI\n",
+    "from ray import serve\n",
+    "\n",
+    "app = FastAPI()\n",
+    "\n",
+    "@serve.deployment\n",
+    "@serve.ingress(app)\n",
+    "class FastAPIDeployment:\n",
+    "    # FastAPI will automatically parse the HTTP request for us.\n",
```

Comment:
```suggestion
    "    # FastAPI will automatically parse the HTTP request for us.\n",
    "    # Because "name" is specified as an argument, it'll be parsed from the query params of the request.\n",
```

Reply:
Maybe we should link out to fastapi docs?